modules/aws/master/master-iam.tf

```diff
@@ -73,6 +73,28 @@ resource "aws_iam_role_policy" "master" {
        "arn:${var.arn_region}:s3:::*"
      ]
     },
+    {
+        "Effect": "Allow",
+        "Action": [
+            "s3:ListBucket",
+            "s3:PutObject",
+            "s3:GetObject",
+            "s3:DeleteObject" 
+        ],
+        "Resource": [
+            "arn:${var.arn_region}:s3:::${var.aws_account}-${var.cluster_name}-g8s-loki",
+            "arn:${var.arn_region}:s3:::${var.aws_account}-${var.cluster_name}-g8s-loki/*"
+        ]
+    },
+    {
+        "Effect": "Allow",
+        "Action": [
```

Comment:
why no resource filter here?

Reply:
this is about S3 access points, not s3 buckets itself. Currently we are not using any Access Points in our VPCs so I cannot filter by anything.

Its mentioned by loki that these permissions are needed, it might fail if we don't add it.